app/services/fees/estimate_instant_service.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+module Fees
+  class EstimateInstantService < BaseService
+    def initialize(organization:, params:)
+      @organization = organization
+      @params = params
+
+      super
+    end
+
+    def call
+      Events::ValidateCreationService.call(organization:, params:, customer:, result:)
+      return result unless result.success?
+
+      if charges.none?
+        return result.single_validation_failure!(field: :code, error_code: 'does_not_match_an_instant_charge')
```

Comment:
No, but since it's not used for an ActiveRecord validation, we have no need for it, the service will only return the error code. And since this service is only intended for API usage, it enought :)